src/lib/components/Feedback/index.js

```diff
@@ -6,18 +6,51 @@ import {BaseElement} from '../BaseElement';
 import {trackEvent} from '../../analytics';
 import './_styles.scss';
 
+/** @types {WebFeedback.Question[]} */
+const questions = [
+  {text: 'Was this page helpful?', label: 'helpful', name: 'Helpfulness'},
+  {
+    text: 'Did this page help you complete your goal(s)?',
+    label: 'effective',
+    name: 'Effectiveness',
+  },
+  {
+    text: 'Did this page have the information you needed?',
+    label: 'complete',
+    name: 'Completeness',
+  },
+  {
+    text: "Was this page's information accurate?",
+    label: 'accuracy',
+    name: 'Accuracy',
+  },
+  {text: 'Was this page easy to read?', label: 'readable', name: 'Readability'},
+];
+
+/** @types {WebFeedback.Question[]} */
+const conditionalQuestions = [
+  {text: 'Does this API meet your needs?', label: 'api', name: 'API'},
+];
+
 /**
  * Element that handles feedback on posts to submit to Analytics.
  *
  * @extends {BaseElement}
  * @final
  */
 class Feedback extends BaseElement {
+  static get properties() {
+    return {
+      conditionals: {type: String},
```

Comment:
It seems like you want to treat this as Array-like data. You can use a [LitElement converter](https://lit-element.polymer-project.org/guide/properties#conversion) to do that. I think that'll be tidier.

```js
return {
  conditionals: {
    converter: (value) => value.split(','),
  },
};
```